src/deprecations.jl

```diff
@@ -3,3 +3,4 @@
 @deprecate InstanceNorm(λ, β, γ, μ, σ², ϵ, momentum) InstanceNorm(λ, β, γ, μ, σ², ϵ, momentum, nothing)
 @deprecate BatchNorm(λ, β, γ, μ, σ², ϵ, momentum) BatchNorm(λ, β, γ, μ, σ², ϵ, momentum, nothing)
 @deprecate GroupNorm(G, λ, β, γ, μ, σ², ϵ, momentum) GroupNorm(G, λ, β, γ, μ, σ², ϵ, momentum, nothing)
+@deprecate outdims(f, inputsize; padbatch=false) outputsize(f, inputsize; padbatch=padbatch)
```

Comment:
```suggestion
@deprecate outdims(f, inputsize) outputsize(f, inputsize)
```
since `padbatch` was not defined before this PR.  The deprecation is not 100% correct, since it is difficult to replicate the behavior of the old `outdims` with the new version, but this is good enough